docs/src/assets/scss/print.scss

```
@@ -184,8 +183,8 @@ button,
     display: none;
 }
 
-a[href^='http']:not([href*='mywebsite.com'])::after {
-    content: ' ('attr(href) ')';
+a[href^="http"]:not([href*="mywebsite.com"])::after {
```

Guessing this might need to be the actual site domain
```suggestion
a[href^="http"]:not([href*="eslint.org"])::after {
```

---

Hehe. Definitely seems that way.